cpp/src/ray/runtime/abstract_ray_runtime.cc

```diff
@@ -115,6 +115,35 @@ std::vector<bool> AbstractRayRuntime::Wait(const std::vector<std::string> &ids,
   return object_store_->Wait(StringIDsToObjectIDs(ids), num_objects, timeout_ms);
 }
 
+rpc::Address GetOwnershipInfo(const std::string &object_id_str) {
+  if (ConfigInternal::Instance().run_mode == RunMode::SINGLE_PROCESS) {
+    return {};
+  }
+  ray::internal::outer_object_id = object_id_str;
+  auto object_id = ray::ObjectID::FromBinary(object_id_str);
+  rpc::Address address;
+  std::string serialized_object_status;
```

Comment:
Could you also leverage `serialized_object_status`? We're trying to support this in Java. See https://github.com/ray-project/ray/pull/19984